Makefile

```diff
@@ -131,3 +131,7 @@ docker-mesh-fluent-bit:
 .PHONY: docker-mesh-bridge
 docker-mesh-bridge:
 	docker build . -t 0xorg/mesh-bridge -f ./dockerfiles/mesh-bridge/Dockerfile
+
```

Comment:
Style Nit: We usually put two newlines between commands (Alex B. introduced this convention a while ago).